test/gtest/stringify.cpp

```diff
@@ -161,14 +161,18 @@ static auto dupModuleText = R"wasm(
    )
   )wasm";
 
+std::vector<uint32_t> hashStringifyModule(Module* wasm) {
+  HashStringifyWalker stringify = HashStringifyWalker();
+  stringify.walkModule(wasm);
+  return std::move(stringify.hashString);
```

Comment:
Is the `move` here just for efficiency? Then as this is test code it's probably fine without it to keep things simpler.

(If we did care about speed we could also use `move` below in `repeatSubstrings` but it feels unnecessary in both places, to me.)

Reply:
ah, thanks for noticing! I do want it in both places :)

Reply:
oh, interesting. when I do std::move in repeatSubstrings, I get `error: moving a local object in a return statement prevents copy elision`